ocaml/xapi/xapi_sr.ml

```diff
@@ -354,13 +361,13 @@ let maybe_copy_sr_rrds ~__context ~sr =
    each PBD got unplugged. *)
 let unload_metrics_from_memory ~__context ~sr =
   let short_uuid = String.sub (Db.SR.get_uuid ~__context ~self:sr) 0 8 in
-  let is_sr_metric = Astring.String.is_suffix ~affix:short_uuid in
```

Comment:
I'd prefer using directly `endswith` here for the single use, there's less scaffolding code and the translation is very easy to understand.